module/VuFind/src/VuFind/Db/Entity/SearchEntityInterface.php

```diff
@@ -40,4 +42,170 @@
  */
 interface SearchEntityInterface extends EntityInterface
 {
+    /**
+     * Get identifier (returns null for an uninitialized or non-persisted object).
+     *
+     * @return ?int
+     */
+    public function getId(): ?int;
+
+    /**
+     * Get user.
+     *
+     * @return UserEntityInterface
+     */
+    public function getUser(): int;
+
+    /**
+     * Set user.
+     *
+     * @param int $user User
+     *
+     * @return SearchEntityInterface
+     */
+    public function setUser(UserEntityInterface $user): SearchEntityInterface;
+
+    /**
+     * Get session id.
```

Comment:
You say "identifier" in the User class but "id" in the interface here. Neither is wrong, but it probably makes sense to make them consistent.